forge-gui/src/main/java/forge/gamemodes/limited/DraftPack.java

```diff
@@ -0,0 +1,170 @@
+package forge.gamemodes.limited;
+
+import forge.item.PaperCard;
+
+import java.util.*;
+
+public class DraftPack implements List<PaperCard> {
```

Comment:
this looks like something to better implement with decorator pattern
see TargetChoices class for clean approach

Reply:
Ah good. Ruby has delegate but I couldn't figure out what it was called in Java